src/module/ETHRates.sol

```diff
@@ -78,7 +78,7 @@ contract ETHRates is IRateProvider {
     }
 
     function _getStETHRate() internal view returns (uint256) {
-        return IStETH(STETH).getSharesByPooledEth(1e18);
+        return IStETH(STETH).getPooledEthByShares(1e18);
```

Comment:
this looks correct now, matches what we have for ynEigen

https://github.com/yieldnest/yieldnest-protocol/blob/main/src/ynEIGEN/LSDRateProvider.sol#L52